src/core/Akka/Actor/ActorCell.DefaultMessages.cs

```diff
@@ -449,6 +449,8 @@ private void Create(Exception failure)
                 CheckReceiveTimeout();
                 if (System.Settings.DebugLifecycle)
                     Publish(new Debug(Self.Path.ToString(), created.GetType(), "Started (" + created + ")"));
+                if(System.Settings.EmitActorTelemetry)
+                    System.EventStream.Publish(new ActorStarted(Self, Props.Type));
```

Comment:
Emit actor started events here - we validate via our specs that these aren't duplicated when an actor restarts.